text/0000-raw-txn-header.md

```diff
@@ -0,0 +1,96 @@
+- Feature Name: send_serialize_txn
+- Start Date: 2018-08-03
+- RFC PR:
+- Sawtooth Issue:
+
+# Summary
+[summary]: #summary
+
+Requesting to add API to the sdk - txn.get_serialized_header(), that will return
+the transaction header bytes.
+
+# Motivation 
+[motivation]: #motivation
+
+For security reasons transaction processor would like to verify the incoming
+transaction header, and signature.  For example, in the private ledger TP (C++)
+that runs inside SGX enclave, we can't trust the signature verification done by
+sawtooth since it happens outside of SGX.  In order to verify the transaction
+signature, TP needs to get the serialized transaction header and not just all
+the fields.
+
+
+# Guide-level explanation
```

Comment:
Either in the guide or reference explanation, we should have an example of the API change (maybe by showing the lines required within apply() to get to the raw header. Ideally, this example would be given in Rust but Python would be okay as well.